### PR TITLE
Support GLO L2P in TGD computation

### PR DESCRIPTION
**Pull request by scarcanague**

### CODE REVIEW

File: src/ephemeris.c [hunk 1]
```diff
@@ -836,22 +836,22 @@ u8 ephemeris_params_valid(const gps_time_t *bgn,
...
-    [7] = 32.0f,
-    [8] = 64.0f,
-    [9] = 128.0f,
-    [10] = 256.0f,
-    [11] = 512.0f,
-    [12] = 1024.0f,
-    [13] = 2048.0f,
-    [14] = 4096.0f,
-    [15] = 6144.0f,
+        [0] = 2.0f,
```
Comment (scarcanague) [inline src/ephemeris.c]: clang formatting

File: src/ephemeris.c [hunk 2]
```diff
@@ -1318,10 +1318,13 @@ double get_tgd_correction(const ephemeris_t *eph, const gnss_signal_t *sid) {
...
         /* The first TGD correction is for the (E1,E5a) combination */
         return gamma * eph->kepler.tgd_gal_s[0];
-      } else {
+      } else if (CODE_GAL_E7I == sid->code || CODE_GAL_E7Q == sid->code ||
+                 CODE_GAL_E7X == sid->code) {
         /* The clock corrections from INAV are for the (E1,E5b) combination, so
          * use the matching group delay correction for all the other signals */
         return gamma * eph->kepler.tgd_gal_s[1];
+      } else {
+        log_error_sid(*sid, "TGD not applied for the signal");
```
Comment (scarcanague) [inline src/ephemeris.c]: If we ever support E6...

File: src/ephemeris.c [hunk 3]
```diff
@@ -1298,9 +1298,9 @@ double get_tgd_correction(const ephemeris_t *eph, const gnss_signal_t *sid) {
          d_tau = t_f2 - t_f1 -> t_f1 = t_f2 - d_tau.
          As clock_err is added to pseudorange,
          d_tau has to be applied with negative sign. */
-      if (CODE_GLO_L1OF == sid->code) {
+      if (CODE_GLO_L1OF == sid->code || CODE_GLO_L1P == sid->code) {
         return 0.0;
-      } else if (CODE_GLO_L2OF == sid->code) {
+      } else if (CODE_GLO_L2OF == sid->code || CODE_GLO_L2P == sid->code) {
```
[the remark this answers is not in the document]
Comment (scarcanague) [reply src/ephemeris.c]: Well, there is nothing in the ICD as it is the military signal :-). Dont know how these Australian stations track it but I would assume bias is very similar. We want to drop the L2P signal anyway inside the PPP filter when we have both LXOF and LXP (Ill put up a PR for this)

File: src/ephemeris.c (hunk 2 above)
Comment (nsirola) [reply src/ephemeris.c]: This now asserts for GAL E1 signals that previously took the default value

**Pull request merged**